bitcoin/src/address/mod.rs

```diff
@@ -49,8 +50,6 @@ use crate::crypto::key::{
 use crate::network::{Network, NetworkKind};
 use crate::prelude::*;
 use crate::taproot::TapNodeHash;
-
-use self::error::P2shError;
```

Comment:
This PR still has formatting changes in it, can we remove them please?

Reply:
Oh oops, I missed your instructions to remove the formatting patch. Let me try again.

Reply:
Yeah sorry man, this whole formatter thing is an unsolved problem - we are trying our best to keep everyone happy, its no easy task.

Reply:
No worries ;) tricky problem when not using rustup.

Force pushed again, diff much more manageable now.